dango/httpd/src/graphql/types/account.rs

```diff
@@ -0,0 +1,89 @@
+use {
+    super::user::User,
+    async_graphql::*,
+    chrono::{DateTime, TimeZone, Utc},
+    dango_indexer_sql::entity,
+    indexer_httpd::context::Context,
+    sea_orm::{ModelTrait, sqlx::types::uuid},
+    serde::Deserialize,
+};
+
+#[derive(Enum, Copy, Clone, Debug, Deserialize, Eq, PartialEq, Hash)]
+pub enum AccountType {
+    Spot,
+    Margin,
+    Multi,
+}
```

Comment:
Can you reuse `dango_types::account_factory::AccountType` instead? I suppose we're missing the `async_graphql::Enum` trait? We can derive this in the dango-types crate (behind an optional feature).